packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.ts

```diff
@@ -8,14 +8,23 @@ import { reads } from 'macro-decorators';
 import { TokenSymbol } from '@cardstack/web-client/utils/token';
 import { WorkflowCardComponentArgs } from '@cardstack/web-client/models/workflow/workflow-card';
 import { currentNetworkDisplayInfo as c } from '@cardstack/web-client/utils/web3-strategies/network-display-info';
+import { TransactionReceipt } from 'web3-core';
+import { task } from 'ember-concurrency-decorators';
+import { taskFor } from 'ember-concurrency-ts';
+import { TaskGenerator } from 'ember-concurrency';
 
 class CardPayDepositWorkflowTransactionStatusComponent extends Component<WorkflowCardComponentArgs> {
+  totalBlockCount = 12;
```

Comment:
Perhaps this value should come from the SDK @habdelra @burieberry?

Reply:
good point. let's add that as a new constant in the SDK for kovan and mainnet

Reply:
> good point. let's add that as a new constant in the SDK for kovan and mainnet (it's a layer 1 artifact)

So I can add it to the `constants.ts` file in the SDK then. Did we pick this number 12 or is it a common thing?

Also, should I be adding the `waitUntilBlock` export to the sdk's `index.ts` with the other exports? Maybe something like `export * from './sdk/utils/general-utils';`? I noticed there's a `waitUntilTransactionMined` fn that we're not currently using in `web-client` but that we could be using.

Reply:
The number 12 came from here https://docs.tokenbridge.net/eth-xdai-amb-bridge/about-the-eth-xdai-amb.

And yes, you should add the `waitUntilBlock` to the exports in the main `index.ts` file. I think we should actually be explicit about which functions we want to export instead of using a `*`, since this is basically our API and we should consciously be deciding which things we want to expose from it. so if you'd like to use the `waitUntilTransactionMined` from the SDK, then let's add that as an explicit export too.